data/cognates/Collecting_cognates_dataset.ipynb

```diff
@@ -0,0 +1,489 @@
+    "            for elem in word_list:\n",
+    "                child_page = \"https://cognates.ru\" + elem[0]\n",
+    "                cognates = get_children(child_page)\n",
+    "                for i in range(min(MAXN, len(cognates))):\n",
+    "                    cognate = cognates[i]\n",
+    "                    first_word.append(elem[1])\n",
+    "                    second_word.append(cognate[1])\n",
+    "                    writer = csv.writer(file)\n",
+    "                    writer.writerows([[elem[1], cognate[1]]])\n",
+    "print(\"Writing complete\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>first_word</th>\n",
+       "      <th>second_word</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>А</td>\n",
+       "      <td>А-конто</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>А</td>\n",
+       "      <td>Аканье</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>А</td>\n",
+       "      <td>Акать</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>А</td>\n",
+       "      <td>Акающий</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>А</td>\n",
+       "      <td>Ашка</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "  first_word second_word\n",
+       "0          А     А-конто\n",
+       "1          А      Аканье\n",
+       "2          А       Акать\n",
+       "3          А     Акающий\n",
+       "4          А        Ашка"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "data = pd.read_csv('cognates_full.csv', encoding='utf_8')\n",
+    "data.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def dist_Levenshtein(s1, s2):\n",
+    "    d = [[0] * (len(s2) + 1) for i in range(len(s1) + 1)]\n",
```

Comment:
In list comprehension we can use `_` instead of `i` :)

As an alternative two-dimension array initialization: `d = [[0] * (len(s2) + 1)] * (len(s2) + 1)`.